SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -11218,17 +11218,30 @@ void CEulerSolver::BC_Inlet(CGeometry *geometry, CSolver **solver_container,
 
         /*--- Total properties have been specified at the inlet. ---*/
 
-        case TOTAL_CONDITIONS:
-
+        case TOTAL_CONDITIONS: case SURFACE_OUTLET:
+          
           /*--- Retrieve the specified total conditions for this inlet. ---*/
-
+          
           if (gravity) P_Total = config->GetInlet_Ptotal(Marker_Tag) - geometry->node[iPoint]->GetCoord(nDim-1)*STANDARD_GRAVITY;
           else P_Total  = config->GetInlet_Ptotal(Marker_Tag);
           T_Total  = config->GetInlet_Ttotal(Marker_Tag);
           Flow_Dir = config->GetInlet_FlowDir(Marker_Tag);
+          
+          if ((Kind_Inlet == SURFACE_OUTLET) && (config->GetExtIter() != 0)) {
+            
+            /*--- Retrieve the specified total conditions for this inlet. ---*/
+            
+            if (gravity) P_Total = config->GetSurface_TotalPressure(0) - geometry->node[iPoint]->GetCoord(nDim-1)*STANDARD_GRAVITY;
+            else P_Total  = config->GetSurface_TotalPressure(0);
```

Comment:
Why is the Marker_Tag in the calls 

- config->GetSurface_TotalPressure(0)
- config->GetSurface_TotalTemperature(0)

forced to be 0?